docs/source/gbd2019_models/causes/neoplasms/multiple_myeloma_2/cancer_model.rst

```diff
@@ -450,117 +482,30 @@ baseline hazard rates at the individual level according to individual simulant
-transform of the martingale residual) to examine any influential observations 
-or outliers.
-
-To check the performance of Cox's model, we will include goodness of fit in our 
-survival analysis results. Specifically, Cox-Snell residuals will be used to assess 
-a model's goodness-of-fit. By plotting the Cox-Snell residual against the cumulative 
-hazard function a model's fit can be assessed. We might modify the standard Cox-Snell 
-residuals to account for the censored observations.
-
-Input Data Table
-++++++++++++++++
-
-.. list-table:: Combination of different observations
-   :header-rows: 1
-   
-   * - Age
-     - Sex
-     - Race
-     - CKD
-     - Cytogenetic risk
-     - Transplantation
-     - Treatment
-     - Duration
-     - Event
-   * - 15 to 95 plus with 5-year age bin
-     - ['Male', 'Female']
-     - ['Black/African', 'Non-Black/African']
-     - ['Stage 1', 'Stage 2', 'Stage 3', 'Stage 4', 'Stage 5']
-     - ['High-risk', 'Standard-risk']
-     - ['Eligible', 'Ineligible']
-     - ['First line not Isa', 'Second line not Isa', 'Third or later line not Isa', 'Isatuximab']
-     - ['Duration from MM to RRMM', 'Duration from MM to death', 'Duration from Nth relapse to (N+1)th relapse', 'Duration from Nth relapse to death']
-     - ['Event of transition from MM to RRMM', 'Event of transition from Nth relapse to (N+1)th relapse', 'Event of death from MM', 'Event of death from Nth relapse']
-
-.. todo::
-
-   Add more details
+#. This cause model assumes no recovery from MM and RRMM since myeloma is an
+   incurable disease. Patients with MM will inevitably develop relapse and the
+   health outcomes worsen with every relapse and line of treatment.
+#. This cause model assumes that relapse and new treatment lines always correspond and
+   occur simultaneously.
+#. This cause model assumes that the GBD incidence rate corresponds to the incidence
+   of symptomatic MM. That said, we are comfortable using GBD incidence of MM
+   as the detection rate of symptomatic MM cases. The incidence of RRMM will be
+   calculated from survival regression analysis based on Flatiron data.
+#. The asymptomatic/idolent state (smoldering MM) is excluded from this cause
+   model because we are not interested in the screening and early managment for
+   MM. As a result, the simulation will not track/model simulants with asymptomatic
+   condition.
+#. YLLs are substantially larger than YLDs for this cause. For now, we will not
+   build a disability component to capture those secondary outcomes.
+#. Based on available data, the most advanced disease state in cause model is
+   fourth or higher relapse of RRMM. We intend to track deaths from simulants
+   who have developed fourth relapse and received fifth-line of treatment but ignore
+   the incident cases from fourth relapse to higher relapse of RRMM. As a result,
+   we will not calculate progress-free survival among simulants with fourth or
```

Comment:
I'm assuming progress-free here = progression-free?

Reply:
Yes, although it should really be "relapse-free" -- I think I missed a few things in this limitations section that should have been updated from Phase 1. Thanks for pointing this out, I will fix 👍